packages/reactivity/src/effect.ts

```
@@ -1,10 +1,48 @@
 import { extend, hasChanged } from '@vue/shared'
 import type { ComputedRefImpl } from './computed'
 import type { TrackOpTypes, TriggerOpTypes } from './constants'
-import { type Link, globalVersion } from './dep'
+import { type Link, globalVersion, type Dep } from './dep'
```

ESLint

[error] Member 'Dep' of the import declaration should be sorted alphabetically.

(sort-imports)